reference/api/overview.md

```diff
@@ -38,27 +38,25 @@ Please read about [security keys](/learn/security/master_api_keys.md) and [how t
 
 ## Errors & status code
 
-#### Success
-
-**200 - Ok**: Everything worked as expected.
-
-**201 - Created**: The resource has been created (synchronous)
-
-**202 - Accepted**: The task has been added to the queue (asynchronous)
-
-**204 - No Content**: The resource has been deleted or no content has been returned
-
-**205 - Reset Content**: All the resources have been deleted
-
-#### Error
-
-**400 - Bad Request**: The request was unacceptable, often due to missing a required parameter.
-
-**401 - Unauthorized**: No valid API key provided.
-
-**403 - Forbidden**: The API key doesn't have the permissions to perform the request.
-
-**404 - Not Found**: The requested resource doesn't exist.
+| Status code | Description                                                                                |
+|-------------|--------------------------------------------------------------------------------------------|
+| 200         | ✅ **Ok** Everything worked as expected.                                                   |
+| 201         | ✅ **Created** The resource has been created (synchronous)                                 |
+| 202         | ✅  **Accepted** The task has been added to the queue (asynchronous)                       |
+| 204         | ✅ **No Content** The resource has been deleted or no content has been returned            |
+| 205         | ✅ **Reset Content** All the resources have been deleted                                   |
+| 400         | ❌ **Bad Request** The request was unacceptable, often due to missing a required parameter |
+| 401         | ❌ **Unauthorized** No valid API key provided                                              |
+| 403         | ❌ **Forbidden** The API key doesn't have the permissions to perform the request           |
+| 404         | ❌ **Not Found** The requested resource doesn't exist                                      |
+
+### Error types
```

Comment:
I think the "Error types" section requires a bit more context. These errors are related but separate from 4xx HTTP status codes. What do you think about something like this:

```suggestion
### Errors

Failed requests are always accompanied by a JSON-formatted error response. Meilisearch errors can be of one of the following types:
```